Sources/FluentUI_iOS/Components/Tab Bar/TabBarItemView.swift

```diff
@@ -300,17 +304,25 @@ class TabBarItemView: UIControl, TokenizedControl {
     }
 
     private func updateColors() {
+        let usesGlassEffectColors = style == .glass
         if isEnabled {
             // We cannot use UIColor(patternImage:) for the tintColor of a UIView. Instead, we have to
             // fully replace the image, so we should not re-tint it here when we have a gradient.
             let shouldTint = isSelected && gradient == nil
             let tintColor = tokenSet[.selectedColor].uiColor
-            titleLabel.textColor = shouldTint ? tintColor : tokenSet[.unselectedTextColor].uiColor
-            imageView.tintColor = shouldTint ? tintColor : tokenSet[.unselectedImageColor].uiColor
+            if usesGlassEffectColors {
+                titleLabel.textColor = shouldTint ? tintColor : tokenSet[.glassUnselectedColor].uiColor
```

Comment:
it we use a switch style() in the tab bar token set, we can avoid a new `glassUnselectedColor` token (and potentially duplicating every token we have for the glass style)

Reply:
Ohh I see. Something like what we do for ButtonTokenSet where we init with a style